extensions/classifiers/PythonProgramTokenizer.js

```diff
@@ -185,24 +185,20 @@ oppia.factory('PythonProgramTokenizer', [
               contstr = '';
               needcont = 0;
               contline = null;
-            }
-
-            else if (
+            } else if (
               needcont && line.slice(-2) !== '\\\n' ||
               line.slice(-3) !== '\\\r\n') {
               tokenizedProgram.push(
                 [PythonProgramTokenType.ERRORTOKEN, contstr + line]);
               contstr = '';
               contline = null;
               continue;
-            }
-
-            else {
+            } else {
               contstr = contstr + line;
               contline = contline + line;
               continue;
             }
-          }
+          } // eslint-disable-line brace-style
```

Comment:
@seanlip: I had to disable check for this line since there is a comment on the next line due to which `else if` cannot be put on the same line as `}`. Ditto below.  
Please suggest what should be done here. We can remove the comments though.  
Thanks!

Reply:
You can put "// New statement." one line below the "else if". It describes what's in the branch so I think that would be fine.

Reply:
Done, thanks!